src/qt/gui_tabMain.py

```diff
@@ -96,36 +96,36 @@ def insert_mn_list(self, name, ip, port, row=None):
         ##--- Details button
         self.btn_details[name] = QToolButton()
         self.btn_details[name].setIcon(self.details_icon)
-        self.btn_details[name].setToolTip('Check STATUS details of masternode "%s"' % name)
+        self.btn_details[name].setToolTip('Check status of "%s"' % name)

```

Comment:
this is the magnifying glass. it is for *details*.
The next button is just for *status*